examples/06-verif-manual/vm-002-beam_stresses_and_deflections.py

```diff
@@ -0,0 +1,189 @@
+###############################################################################
+# Define Material
+# ~~~~~~~~~~~~~~~
+# Set up the material and its type (a single material, with a beam-type section).
+
+mapdl.mp("EX", 1, 30E6)
+mapdl.mp("PRXY", 1, 0.3)
+
+###############################################################################
+# Define Section
+# ~~~~~~~~~~~~~~
+# Set up the cross-section for a beam element.
+
+w_f = 1.048394965
+w_w = 0.6856481
+sec_num = 1
+mapdl.sectype(sec_num, "BEAM", "I", "ISection")
+mapdl.secdata(15, 15, 28+(2*w_f), w_f, w_f, w_w)
+
+
+###############################################################################
+# Define Geometry:
+# ~~~~~~~~~~~~~~~~~~~~~~
+# Set up the nodes and elements.  This creates a mesh just like
+# in the problem setup. We create a square of nodes and use `fill` to add
+# mid-point nodes to two opposite sides.
+
+# Define nodes
+mapdl.n(1)
+mapdl.n(5, 480)
+mapdl.n(6, 60, "1 $ N", 10, 420, 1)
+
+print(mapdl.mesh.nodes)  # list the node coordinates
+
+print(mapdl.mesh.nnum)  # list the node numbers
+
+mapdl.fill(1, 5)  # Generates a line of nodes between two existing nodes.
+mapdl.fill(6, 10)
+mapdl.e(1, 2, 6)  # Define an element by node connectivity.
+mapdl.egen(4, 1, 1)  # Generates elements from an existing pattern.
+mapdl.eplot(show_node_numbering=True, line_width=5, cpos="xy")  # Display elements with their nodes numbers.
+
+
+###############################################################################
+# Define Boundary Conditions
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Fix the nodes at the larger end (the "wall" end) and apply a vertical force
+# to the whole structure.
+
+mapdl.d(2, "UX", lab2="UY")  # Application of boundary conditions.
+mapdl.d(4, "UY")
+mapdl.nsel("S", "LOC", "Y", 0)
+mapdl.d("ALL", "UZ")
+mapdl.d("ALL", "ROTX")
+mapdl.d("ALL", "ROTY")
+mapdl.nsel("ALL")
+
+
+###############################################################################
+# Define Distributed Loads
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~
+# - Apply a distributed force of :math:`w = (10000/12) lb/in` in the y-direction
+
+mapdl.sfbeam(1, 1, "PRES", 10000/12)
+mapdl.sfbeam(4, 1, "PRES", 1E4/12)
+mapdl.finish()
+
+
+###############################################################################
+# Solve
+# ~~~~~
+# Enter solution mode and solve the system.
+
+mapdl.run("/SOLU")
+out = mapdl.solve()
+print(out)
+mapdl.finish()
+
+
+###############################################################################
+# Post-processing
+# ~~~~~~~~~~~~~~~
+# Enter post-processing. Select the nodes at ``y=10`` and ``y=0``, and
+# sum the forces there. Then store the y-components in two variables:
+# ``reaction_1`` and ``reaction_2``.
+
+mapdl.post1()  # Start Post-Processing
+mapdl.set(1)  # Select first Load Step
+
+# Define Maximum Stress
+
+
+a = mapdl.post_processing.nodal_elastic_eqv
+
+
+# print(seqv_max)
+
+# # Difine deflection at the middle of the beam
+# mid_node_uy = mapdl.get('disp', 'node', 3, 'u', 'y')
```

Comment:
Use [get_value](https://mapdldocs.pyansys.com/api/_autosummary/ansys.mapdl.core.Mapdl.get_value.html) instead as this returns either a float or an int when successful.